selfdrive/car/volkswagen/interface.py

```diff
@@ -36,6 +43,11 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=None):
       else:  # No trans message at all, must be a true stick-shift manual
         ret.transmissionType = TransmissionType.manual
 
+      if 0x86 in fingerprint[1]:  # LWI_01 seen on bus 1, we're wired to the CAN gateway
```

Comment:
is this reliable with the current minimum fingerprinting time?

Reply:
let's hardcode the network location to the camera until the follow up PR

Reply:
> is this reliable with the current minimum fingerprinting time?

Yes, it's a 100Hz message that seems to be available immediately upon openpilot startup in CAN logs. This is one of the major reasons we had powertrain added to bus 1 on gateway harnesses. Back in the day, I first tried to sense bus position by watching for object fusion messages on the *camera* harness, and those didn't show up quite fast enough, but this was way before CAN message collection window ran to 100ms for UDS fingerprinted cars.

Reply:
> let's hardcode the network location to the camera until the follow up PR

I don't think that's the right thing to do:

* This PR, as it stands alone, gateway integrated but with a factory LKAS camera, is a perfectly valid config to run and drive with. It's how I typically run my Golf R and Tiguan, and it's how many other folks are set up as well. It would be technically valid to accept this PR and reject the camera-optional PR.

* For the follow-up PR to make the camera optional, I actually do have some things to prove around fingerprint time. I think it'll be okay, and if it isn't I know how to solve it easily, but I intend to script up something to look at logs from several other MQB user devices before I move forward. Feel free to make me show my work on that one.

Reply:
fair enough, should be good to merge after addressing the other comment